src/apps/admin/features/company.create/helpers/renderExpansionPanel.js

```diff
@@ -14,10 +14,14 @@ export function CodeXExpansionPanel({
   titleComponent,
   titleStyle,
   titleVariant,
+  error,
```

Comment:
Not sure I agree, I see it with the border we can style that but if we want to do something fancier, like you said maybe an exclamation point by the name, we're going to have to handle that sort of addition within the component itself. By passing an error state it sort of future proofs it so we can make that error state be whatever we want and we only have to implement a custom error state once in the expansion component. Then it will be extended to all expansions that utilize form validation if we put form validation into the rest of the form.